addons/sale_mrp/tests/test_sale_mrp_report.py

```diff
@@ -69,60 +68,3 @@ def test_deliver_and_invoice_tracked_components(self):
             'account.report_invoice_with_payments', invoice.ids)[0]
         text = html2plaintext(html.decode())
         self.assertRegex(text, r'Product By Lot\n1.00Units\nLOT0001', "There should be a line that specifies 1 x LOT0001")
-
-    def test_report_forecast_for_mto_procure_method(self):
```

Comment:
Why is that test removed? The tested case seems still legit, isn't it?
At least that's what it looked like in the runbot when I tried to run a similar setup:
![image](https://github.com/user-attachments/assets/469a4e77-2997-426b-b134-86615cb2abd5)
Didn't check the specifics, but it seemed still relevant. :thinking: